samples/gltf_viewer.cpp

```diff
@@ -747,14 +747,25 @@ int main(int argc, char** argv) {
                 createJitShaderProvider(engine, OPTIMIZE_MATERIALS) :
                 createUbershaderProvider(engine, UBERARCHIVE_DEFAULT_DATA, UBERARCHIVE_DEFAULT_SIZE);
 
-        app.assetLoader = AssetLoader::create({engine, app.materials, app.names });
+        AssetConfigurationExtended extConfig = {};
```

Comment:
You're not doing anything with `extConfig`. You only set its gltfPath but you don't pass the config to anything.

Reply:
sorry, I did not mean to commit gltf_viewer changes.  That was just for testing.  I undid the changes